dev-db/etcd/files/etcd2.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=etcd2
+
+[Service]
+User=etcd
+PermissionsStartOnly=true
+Environment=ETCD_DATA_DIR=/var/lib/etcd
```

Comment:
`/var/lib/etcd2`

Data directories need to be different. That also means etcd2 needs its own tmpfiles to create the new data dir.